tests/integration/cli/actions/test_select_client_account.py

```diff
@@ -166,24 +177,21 @@ def test_select_client_account_with_balance_display(
     mock_staking_agent,
     mock_token_agent,
     selection,
-    show_staking,
     show_matic,
     stake_info,
 ):
-
     # Setup
     mock_staking_agent.get_all_stakes.return_value = stake_info
 
     # Missing network kwarg with balance display active
-    blockchain_read_required = any((show_staking, show_matic))
+    blockchain_read_required = any((show_matic,))
     if blockchain_read_required:
         with pytest.raises(
             ValueError, match="Pass domain name or registry; Got neither."
```

Comment:
This part of the test is no longer applicable now...can this check be removed?

Reply:
✔️